code/dynamic_programming/src/longest_repeating_subsequence/longest_repeating_subsequence.py

```diff
@@ -0,0 +1,16 @@
+def longestRepeatingSubsequence(str): 
```

Comment:
Use underscore case and reformat the file with black.

Reply:
@arnavb  Done.